Assets/Dev/Si Wai/JumpingMovement.cs

```diff
@@ -3,7 +3,7 @@
 public class JumpingMovement : MonoBehaviour
 {
     public float bounceForceUp = 5.0f;
```

Comment:
Unsure if we want to use public variables everywhere, or if we prefer:
`[SerializeField] private float ` 
but that's something @WantToBeeMe can tell us

Reply:
yea, defenitly `[SerializedField] private`  :)

So step 1 is always think, "should this be private or public" and make it that.
And step 2 is: "should this be in the inspector",   and add `[SerializeField]` or `[HideInInspector]` if this is not the case yet.